src/Microsoft.AspNetCore.Razor/Tokenizer/Internal/CSharpKeywordDetector.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Tokenizer.Symbols;
 
-namespace Microsoft.AspNetCore.Razor.Tokenizer
+namespace Microsoft.AspNetCore.Razor.Tokenizer.Internal
 {
     internal static class CSharpKeywordDetector
```

Comment:
> might as well just keep the internal guys internal and where they were originally [typo corrected]

I'm confused by the `namespace` change here if the decision was to leave `internal` classes where they were. Also, does that decision cover `src` as well as `test`. A few classes in both have moved and become `public`. This and `DisposableAction` moved but aren't `public`.